inlang.config.js

```diff
@@ -0,0 +1,18 @@
+export async function defineConfig(env) {
```

Comment:
This file is probably not in the correct folder?

Reply:
Inlang maybe will support that in the future too, but for now the config file needs to be in the root dir.